resources/qml/SidebarHeader.qml

```diff
@@ -314,6 +314,62 @@ Column
         }
     }
 
+    //Buildplate row separator
+    Rectangle {
+        id: separator
+
+        anchors.leftMargin: UM.Theme.getSize("sidebar_margin").width
+        anchors.rightMargin: UM.Theme.getSize("sidebar_margin").width
+        anchors.horizontalCenter: parent.horizontalCenter
+        visible: buildplateRow.visible
+        width: parent.width - UM.Theme.getSize("sidebar_margin").width * 2
+        height: visible ? UM.Theme.getSize("sidebar_lining_thin").height / 2 : 0
+        color: UM.Theme.getColor("sidebar_lining_thin")
+    }
+
+    //Buildplate row
+    Item
+    {
+        id: buildplateRow
+        height: UM.Theme.getSize("sidebar_setup").height
+        visible: Cura.MachineManager.hasVariantBuildplates && !sidebar.monitoringPrint && !sidebar.hideSettings
```

Comment:
This will be refactored in the machine manager refactor I guess? (e.g. globalVariants and extruderVariants?)

Reply:
Yes, probably.